developers/contracts/liabtc-mint-registry.md

```diff
@@ -0,0 +1,131 @@
+
+#### `set-burn-request`
+
+Creates or modifies burn requests in the [`burn-requests`](#burn-requests) map as specified in the `details` parameter. It is called by the functions [`request-burn`](liabtc-mint-endpoint.md#request-burn), [`revoke-burn`](liabtc-mint-endpoint.md#revoke-burn) and [`finalize-burn`](liabtc-mint-endpoint.md#finalize-burn) in the [`liabtc-mint-endpoint`][mint] contract.
+
+New requests are created by passing `u0` as the `request-id` parameter. On each request creation, the [`burn-request-nonce`](#burn-request-nonce) variable is incremented by one, resulting in the id of the new request.
+
+##### Parameters
+
+| Name         | Type                                                                              |
+| ------------ | --------------------------------------------------------------------------------- |
+| `request-id` | `uint`                                                                            |
+| `details`    | `{ requested-by: principal, amount: uint, requested-at: uint, status: (buff 1) }` |
+
+#### `transfer`
+
+Calls the `transfer` function of the `token-trait` passed with `as-contract` privilege. The caller has the ability to send tokens from the registry's balance to a designated `recipient`.
+
+##### Parameters
+
+| Name          | Type              |
+| ------------- | ----------------- |
+| `amount`      | `uint`            |
+| `recipient`   | `principal`       |
+| `token-trait` | `<sip-010-trait>` |
+
+### Supporting features
+
+#### `is-dao-or-extension`
+
+Standard protocol function to check whether the `contract-caller` is an enabled extension within the DAO or the `tx-sender` is the DAO itself (proposal execution scenario). The enabled extension check is delegated to the LISA's `executor-dao` contract.
+
+### Getters
+
+#### `get-burn-request-nonce`
+
+Returns the [`burn-request-nonce`](#burn-request-nonce) variable.
+
+#### `get-burn-request-or-fail`
+
+Returns the burn request on the [`burn-requests`](#burn-requests) map at a given key. If there is no entry for the provided `request-id`, throws.
+
+#### Parameterss
+
+| Name         | Type   |
+| ------------ | ------ |
+| `request-id` | `uint` |
+
+## Storage
+
+### `burn-request-nonce`
+
+| Data     | Type   |
+| -------- | ------ |
+| Variable | `uint` |
+
+Indicates the `request-id` of the last burn request created. This variable can only monotonically increase. It initialized as `u0`, the key that will always have an empty value.
+
+### `burn-requests`
+
+| Data | Type                                                                                   |
+| ---- | -------------------------------------------------------------------------------------- |
+| Map  | `uint { requested-by: principal, amount: uint, requested-at: uint, status: (buff 1) }` |
+
+Map that stores the burn requests used by the [`liabtc-mint-endpoint`][mint].
```

Comment:
Segun otros docs y comentarios, esto no podría ser usado por otros componentes (que se lo propongan) también? Deberíamos hacer algun comentario sobre esto?  Quizas mencionarlo aca?   Va.. digo.. si es asi.. entonces esa oracion esta incompleta .. liabtc-mint-endpoing and/or others..

Reply:
Ah. ok entiendo! Gracias!